arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -1543,6 +1559,19 @@
 			path = "Viewshed (camera)";
 			sourceTree = "<group>";
 		};
+		12D6602B226697040005DE8A /* Group layers */ = {
+			isa = PBXGroup;
+			children = (
+				12D6602E226697320005DE8A /* GroupLayers.storyboard */,
+				12D66032226697320005DE8A /* GroupLayersSectionView.swift */,
+				12D6602C226697320005DE8A /* GroupLayersSectionView.xib */,
+				12D6602F226697320005DE8A /* GroupLayersViewController.swift */,
+				12D66031226697320005DE8A /* LayersTableViewController.swift */,
+			);
+			name = "Group layers";
+			path = "New Group";
```

Comment:
Please fix the path of the group and the group's members.

![image](https://user-images.githubusercontent.com/2257493/56251000-9be64180-6066-11e9-85a6-5acb7ed3197d.png)

Reply:
done

Reply:
You fixed the path of the group, but not the path of the group's members:

![image](https://user-images.githubusercontent.com/2257493/56297364-0dfd6b80-60e5-11e9-891f-2f9d8e7fd100.png)

Reply:
Interesting. Group member's path look good at my end. 

![Screen Shot 2019-04-17 at 9 29 45 AM](https://user-images.githubusercontent.com/8196343/56304761-a8fd4200-60f3-11e9-8ee6-239d060e4f71.png)

![Screen Shot 2019-04-17 at 9 29 56 AM](https://user-images.githubusercontent.com/8196343/56304764-aac70580-60f3-11e9-87a5-2ac1c6be0c86.png)

Reply:
nevermind. I saw where the problem is.